noir_stdlib/src/uint128.nr

```diff
@@ -0,0 +1,205 @@
+use crate::ops::{Add, Sub, Mul, Div};
+use crate::cmp::{Eq, Ord, Ordering};
```

Comment:
We should implement `Rem` as well since we have `unconstrained_div`.

We can also implement the bitwise operators.

Reply:
I've implemented all the bitwise operations. We could also add the bitshifts but that's a little more complex.